grc/CMakeLists.txt

```diff
@@ -25,9 +25,9 @@ include(GrPython)
 message(STATUS "")
 
 GR_PYTHON_CHECK_MODULE_RAW(
-    "python2 >= 2.7.6 or python3 >= 3.6.5"
+    "python3 >= 3.6.5"
     "import sys; \
-     requirement = (3, 6, 5) if sys.version_info.major >= 3 else (2, 7, 6); \
+     requirement = (3, 6, 5); \
```

Comment:
Any way we can use the `GR_PYTHON_MIN_VERSION` here too? Better to automate this from a single variable than to have to manually tweak this every time a change is needed. IMHO.

Reply:
Absolutely true!

Reply:
Fixed on the other PR.